### PR TITLE
Project Closeout Report

### PR DESCRIPTION
**Pull request by SuganyaAK**
Production grade Dapps - Final closeout report 
---
I have created a draft report based on the template shared, I included the challenge KPIs and Project KPIs based on the documentation we have and the milestones achieved so far.

---
I would appreciate your feedback, review comments and suggestions.

1. **Project Manager** - Please confirm who should be listed as the "Project Manager"
2. **Project Completion Date** - Do we have to include the completion date close to our submission , or should I use the estimated milestone date on our proposal?
3. In our final milestone output, it is mentioned **"Collaboration with Demeter where Smart Contracts are available to launch in their platform"**,What should our next steps be regarding this?
4. In this document, under "Adoption Rate and Usage Metrics," I have included the GitHub insights for the metrics. Is this correct?

### CODE REVIEW

File: pages/closeoutreport.typ [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+/ Project manager :
+
+/ Project Start Date : Oct 8, 2023
+
+/ Project completion Date : 
+
+/ List of challenge KPIs and how the project addressed them :
+
+- *Addressing the Scarcity : * Aimed to reduce the scarcity of openly available, production-level codebases, the project achieved its goal by providing a suite of five versatile, open-source smart contract libraries. These libraries are readily accessible and can be adopted for general use by removing application-specific logic.
```
Comment (nikhils9) [inline pages/closeoutreport.typ]: I think we can rephrase this 
> These libraries are readily accessible and can be adopted for general use by removing application-specific logic.

as
These libraries are readily accessible and can be adopted for general use as it is or by incorporating additional application-specific logic.

File: pages/closeoutreport.typ [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+- *Fostering Ecosystem Innovation :* The five smart contract libraries are a testament to best practices in smart contract design, testing, and optimization. Designed with composability in mind, they serve as foundational blocks for building complex DApps. These libraries are ideal for teams aspiring to transition or start fresh in the Cardano ecosystem, streamlining the development process and fostering innovation within the ecosystem.
```
Comment (nikhils9) [inline pages/closeoutreport.typ]:
```suggestion
- *Fostering Ecosystem Innovation :* The five smart contract libraries are a testament to best practices in smart contract design, testing and optimization. Designed with composability in mind, they serve as foundational blocks for building complex DApps. These libraries are ideal for teams aspiring to transition or start fresh in the Cardano ecosystem, streamlining the development process and fostering innovation within the ecosystem.
```

File: pages/closeoutreport.typ [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+- *Ensure Code Quality and Production-Ready Resources : * The challenge was to uphold high standards of code quality, adherence to best practices, and readiness for production, aimed at alleviating the pain points faced by developers. The quality of the smart contract libraries was assessed through comprehensive methods, including code reviews, unit testing, and property-based testing.
```
Comment (nikhils9) [inline pages/closeoutreport.typ]:
```suggestion
- *Ensure Code Quality and Production-Ready Resources : * The challenge was to uphold high standards of code quality, adherence to best practices, and readiness for production, aimed at alleviating the pain points faced by developers. The quality of the smart contract libraries was assessed through comprehensive methods, including code reviews, unit testing and property-based testing.
```

File: pages/closeoutreport.typ [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+/ List of project KPIs and how the project addressed them :
+
+- *Adoption Rate and Usage Metrics :*  The important statistics of the project's overall GitHub activity can be found at this #link("https://github.com/Anastasia-Labs/production-grade-dapps/pulse/monthly")[link]. This includes all the key repositories of this proposal which are detailed in the extensive documentation found here #link("https://anastasia-labs.github.io/production-grade-dapps")[documentation].
+
+
+- *Extensive Documentation  :* Comprehensive documentation for each smart contract library, including detailed explanations of contract functionality, parameters, and usage, was provided. Detailed diagrams aid in understanding contract architecture and data flows.
+
+
+/ Key achievements (in particular around collaboration and engagement) :
```
Comment (nikhils9) [inline pages/closeoutreport.typ]: Maybe the proposal deliverables can be re-stated here as achievements along with necessary collaboration with Demeter team.

**Comment by nikhils9** [comment]
> ## Production grade Dapps - Final closeout report
> 
> I have created a draft report based on the template shared, I included the challenge KPIs and Project KPIs based on the documentation we have and the milestones achieved so far.
> 
> I would appreciate your feedback, review comments and suggestions.
> 
>     1. **Project Manager** - Please confirm who should be listed as the "Project Manager"

Jonathan?

> 
>     2. **Project Completion Date** - Do we have to include the completion date close to our submission , or should I use the estimated milestone date on our proposal?

Not sure.

> 
>     3. In our final milestone output, it is mentioned **"Collaboration with Demeter where Smart Contracts are available to launch in their platform"**,What should our next steps be regarding this?

I think we will need to achieve this goal before submitting to catalyst reviewers.
> 
>     4. In this document, under "Adoption Rate and Usage Metrics," I have included the GitHub insights for the metrics. Is this correct?

Yes, it seems to be a good solution. However, since this repository is just a Readme with list of actual project repository links, it would be better to provide the links to individual repo's insights.

@MladenLm can you please provide your inputs/ suggestions on the open questions here.

**Comment by nikhils9** [comment]
I'll try answering the below sections expecting improvements/ reviews from others in order to finalize.

/ Key learnings:
- Enhanced understanding of different business domains in which these smart contract projects provide value.
- Improvements in tooling and style of documentation to facilitate easy onboarding of developers.

/ Next steps for the product or service developed:
- Expansion by adding new production grade dapps
- Increasing adoption by making these smart contracts available in different smart contract languages.

Please try and elaborate on above if feasible.

File: pages/closeoutreport.typ (hunk 2 above)
Comment (SuganyaAK) [reply pages/closeoutreport.typ]: Updated the suggested change.

File: pages/closeoutreport.typ (hunk 3 above)
Comment (SuganyaAK) [reply pages/closeoutreport.typ]: Updated.

**Comment by SuganyaAK** [comment]
> > ## Production grade Dapps - Final closeout report
> > I have created a draft report based on the template shared, I included the challenge KPIs and Project KPIs based on the documentation we have and the milestones achieved so far.
> > I would appreciate your feedback, review comments and suggestions.
> > ```
> > 1. **Project Manager** - Please confirm who should be listed as the "Project Manager"
> > ```
> 
> Jonathan?
> 
> > ```
> > 2. **Project Completion Date** - Do we have to include the completion date close to our submission , or should I use the estimated milestone date on our proposal?
> > ```
> 
> Not sure.
> 
> > ```
> > 3. In our final milestone output, it is mentioned **"Collaboration with Demeter where Smart Contracts are available to launch in their platform"**,What should our next steps be regarding this?
> > ```
> 
> I think we will need to achieve this goal before submitting to catalyst reviewers.
> 
> > ```
> > 4. In this document, under "Adoption Rate and Usage Metrics," I have included the GitHub insights for the metrics. Is this correct?
> > ```
> 
> Yes, it seems to be a good solution. However, since this repository is just a Readme with list of actual project repository links, it would be better to provide the links to individual repo's insights.
> I linked to each project's insights as suggested..

> @MladenLm can you please provide your inputs/ suggestions on the open questions here.

**Comment by SuganyaAK** [comment]
> I'll try answering the below sections expecting improvements/ reviews from others in order to finalize.
> 
> / Key learnings:
> 
> * Enhanced understanding of different business domains in which these smart contract projects provide value.
> * Improvements in tooling and style of documentation to facilitate easy onboarding of developers.
> 
> / Next steps for the product or service developed:
> 
> * Expansion by adding new production grade dapps
> * Increasing adoption by making these smart contracts available in different smart contract languages.
> 
> Please try and elaborate on above if feasible.

I have elaborated based on your inputs. Kindly review. However, as per your expectations will keep it open to include any other comments/suggestions from the team.

File: pages/closeoutreport.typ (hunk 1 above)
Comment (SuganyaAK) [reply pages/closeoutreport.typ]: Revised and modified.

**Comment by nikhils9** [comment]
> > I'll try answering the below sections expecting improvements/ reviews from others in order to finalize.
> > / Key learnings:
> > 
> > * Enhanced understanding of different business domains in which these smart contract projects provide value.
> > * Improvements in tooling and style of documentation to facilitate easy onboarding of developers.
> > 
> > / Next steps for the product or service developed:
> > 
> > * Expansion by adding new production grade dapps
> > * Increasing adoption by making these smart contracts available in different smart contract languages.
> > 
> > Please try and elaborate on above if feasible.
> 
> I have elaborated based on your inputs. Kindly review. However, as per your expectations will keep it open to include any other comments/suggestions from the team.

Looks good to me!

**Comment by solidsnakedev** [comment]
> > ## Production grade Dapps - Final closeout report
> > I have created a draft report based on the template shared, I included the challenge KPIs and Project KPIs based on the documentation we have and the milestones achieved so far.
> > I would appreciate your feedback, review comments and suggestions.
> > ```
> > 1. **Project Manager** - Please confirm who should be listed as the "Project Manager"
> > ```
> 
> Jonathan?
> 
> > ```
> > 2. **Project Completion Date** - Do we have to include the completion date close to our submission , or should I use the estimated milestone date on our proposal?
> > ```
> 
> Not sure.
> 
> > ```
> > 3. In our final milestone output, it is mentioned **"Collaboration with Demeter where Smart Contracts are available to launch in their platform"**,What should our next steps be regarding this?
> > ```
> 
> I think we will need to achieve this goal before submitting to catalyst reviewers.
> 
> > ```
> > 4. In this document, under "Adoption Rate and Usage Metrics," I have included the GitHub insights for the metrics. Is this correct?
> > ```
> 
> Yes, it seems to be a good solution. However, since this repository is just a Readme with list of actual project repository links, it would be better to provide the links to individual repo's insights.
> 
> @MladenLm can you please provide your inputs/ suggestions on the open questions here.

yes, you can add me as project manager @SuganyaAK

**Comment by SuganyaAK** [comment]
@MladenLm @solidsnakedev

I understand the importance of including a cover page with logo in a audit report. 
However, considering the closeout report guidelines specify a 2-3 page written summary, 
how crucial is it to add the logo and contents page in the closeout report?

Appreciate your thoughts on this.

closeoutreport_guidelines

File: pages/closeoutreport.typ (hunk 4 above)
Comment (MladenLm) [reply pages/closeoutreport.typ]: Yup, let's go with that

**Comment by MladenLm** [comment]
I think 3 pages is not a hard limit, and I would like to have our branding/logo on the report so it can be recognizable when shared in the community

**Comment by MladenLm** [comment]
1. Please put Jonathan as a project manager
2. Date of completion can be May 31th 
3. Hayden published our contracts on Demeter just today - please link those in the report too - https://demeter.run/ports, https://demeter.run/starter-kits/anastasialabs-linear-vesting
4. That is good. In case we will have to add something, the Catalyst team will tell us

**Comment by SuganyaAK** [comment]
I have updated the content according to your comments and adjusted the page layout to better suit the needs of this report. Please review and let me know if any further suggestions or modifications are needed.

**Review by hadelive — APPROVED** [review]

**Review by nikhils9 — APPROVED** [review]

**Pull request merged**